src/coreclr/scripts/superpmi.py

```diff
@@ -2796,6 +3045,53 @@ def process_base_jit_path_arg(coreclr_args):
 
     raise RuntimeError("No baseline JIT found")
 
+def get_pintools_path(coreclr_args):
+    """ Get the local path where we expect pintools for this OS to be located
+    
+    Returns:
+        A path to the folder.
+    """
+    return os.path.join(coreclr_args.spmi_location, "pintools", pintools_current_version, coreclr_args.host_os.lower())
+
+def get_pin_exe_path(coreclr_args):
+    """ Get the local path where we expect the pin executable to be located
+    
+    Returns:
+        A path to the executable.
+    """
+    root = get_pintools_path(coreclr_args)
+    exe = "pin.exe" if coreclr_args.host_os.lower() == "windows" else "pin"
+    return os.path.join(root, exe)
+
+def get_inscount_pintool_path(coreclr_args):
+    """ Get the local path where we expect the clrjit inscount pintool to be located
+    
+    Returns:
+        A path to the pintool library.
+    """
+    if coreclr_args.host_os.lower() == "osx":
+        pintool_filename = "libclrjit_inscount.dylib"
+    elif coreclr_args.host_os.lower() == "windows":
+        pintool_filename = "clrjit_inscount.dll"
+    else:
+        pintool_filename = "libclrjit_inscount.so"
+
+    return os.path.join(get_pintools_path(coreclr_args), "clrjit_inscount_" + coreclr_args.arch, pintool_filename)
+
+def download_clrjit_pintool(coreclr_args):
+    """ Download the pintool package for doing measurements of the JIT from Azure Storage.
+    """
+
+    if os.path.isfile(get_pin_exe_path(coreclr_args)):
+        return
+
+    pin_dir_path = get_pintools_path(coreclr_args)
+    pintools_rel_path = "{}/{}/{}.zip".format(az_pintools_root_folder, pintools_current_version, coreclr_args.host_os.lower())
```

Comment:
I believe this will be uploaded manually once in a while?

Reply:
Yep, this is just uploaded manually to Azure Storage. I've versioned it to be safe although that probably wasn't necessary.